styles/styles.css

```diff
@@ -716,6 +716,10 @@ main .section.light-blue {
   background-color: var(--transparent-blue-light-color);
 }
 
+main .section.no-padding .section-container {
+  padding: 0;
```

Comment:
Is this needed? I think all `section-containers` must have same styling unless there's a special case to handle.

Reply:
Removed

Reply:
Needed for tablet view (cards block)